src/main/java/net/mcreator/workspace/misc/WorkspaceInfo.java

```diff
@@ -92,11 +92,15 @@ public String getUUID() {
 	}
 
 	public double random(String seed) {
+		return random(seed, 0);
+	}
+
+	public double random(String seed, long offset) {

```

Comment:
Where is this gonna be used?

Reply:
I needed it but found a better solution, but some 3rd party plugins may use it